cookbook/setup_data/customize_dataset.rst

```diff
@@ -1,27 +1,41 @@
-How to Customize the Installation
-=================================
+How to Customize the Dataset
+============================
 
-The Akeneo PIM allows to prepare a custom data set to use during the installation.
+Akeneo PIM ships with two datasets that are located in `vendor/akeneo/pim-community-dev/src/Pim/Bundle/InstallerBundle/Resources/fixtures`:
+* `icecat-demo-dev` to be able to take a look and play with the PIM with already preset families, categories, products, etc..
+* `minimal` to start a brand new project
 
-The idea is to allow you to setup your own catalog structure or your own demo data.
+By default, Akeneo PIM ships with `icecat-demo-dev`.
 
-You can configure the data set in the ``app/config/parameters.yml`` file:
+Switch From Icecat to Minimal
+-----------------------------
 
-.. literalinclude:: ../../app/config/parameters.yml.dist
-   :language: yaml
-   :prepend: # /app/config/parameters.yml
-   :linenos:
+Edit the file `app/config/parameters.yml` and add the line
+`installer_data: 'AcmeInstallerBundle:mydataset'`
+
+.. note::
 
-The following steps allow you to easily define your own basic entities when you install the PIM.
+   If you use the enterprise version please use `PimEnterpriseInstallerBundle:minimal` instead of `PimInstallerBundle:minimal`.
+
+You can now (re)install your database by running:
+
+.. code-block:: bash
+
+    > php app/console pim:installer:db --env=dev
+
+Use Your Own Dataset
+--------------------
+
+The Akeneo PIM allows to prepare a custom data set to use during the installation. The idea is to allow you to setup your own catalog structure or your own demo data. The following steps allow you to easily define your own basic entities when you install the PIM.
```

Comment:
"to allow you" -> repetition with the next sentence. You can drop it, it's not necessary imo.